Docs/Configuration.tex

```diff
@@ -2216,9 +2215,9 @@ \subsection{Quirks Properties}\label{kernelpropsquirks}
   MSR modification in AppleIntelCPUPowerManagement.kext, commonly causing early
   kernel panic, when it is locked from writing.
 
-  Certain firmwares lock \texttt{PKG\_CST\_CONFIG\_CONTROL} MSR register. To check its
-  state one can use bundled \texttt{VerifyMsrE2} tool. Select firmwares have this
-  register locked on some cores only.
+  Certain firmwares lock \texttt{PKG\_CST\_CONFIG\_CONTROL} MSR register. The bundled
+  \texttt{VerifyMsrE2} tool can be used to check its state. Some firmware have this
+  register locked only on some cores.
```

Comment:
*merely* can be more academic than *only*?

Reply:
Explain "merely"?
Changes "locked on some cores only" to "locked only on some cores". I.e., moves "only" to correct position.

Or do you mean to use "merely" in place of "only" which would be wrong as "merely" connotes triviality while "only" connotes singularity which I think is the intention here.

Reply:
Thanks for the explanation! I am not a native English speaker, though I kind of remembered such thesauruses in some academic writing classes... @_@